core/build.gradle

```diff
@@ -101,7 +101,7 @@ dependencies {
   compile 'com.android.support:appcompat-v7:24.2.1'
 
   compile 'com.mapzen.android:lost:2.1.2'
-  compile 'com.mapzen.android:pelias-android-sdk:1.0.0'
+  compile 'com.mapzen.android:pelias-android-sdk:1.1.0-SNAPSHOT'
```

Comment:
You plan to update this to maven central version once `pelias-android-sdk` updates are pushed right?

Same for `on-the-road`